### DISCUSSION TITLE
iTwin display

### DISCUSSION THREAD

**Discussion by Swing366** [iTwin/itwinjs-core#1706] [Q&A]
Hello,guys
A iModel display in a view. look at the below picture. Could you give some guidance about why the display of the same slab in the 2nd screenshot has changed.
PS: the slab contain the concrete and rebar.
Thanks in advance!!
![Nearly to look](https://user-images.githubusercontent.com/85770243/123191972-9dc4ae80-d4d4-11eb-837b-843b5b911e26.jpg)
![Be far to look ](https://user-images.githubusercontent.com/85770243/123192047-baf97d00-d4d4-11eb-949a-ebaa60631591.jpg)

**Comment by pmconne** [comment]
Difficult to diagnose with the information provided. If you want, send the .bim file to pc4868696@gmail.com.
Alternatively (or additionally), provide more information, such as: what is this app? Is it viewing a local copy of the iModel or one in the cloud? How was the iModel produced? Etc.
